script.js

```diff
@@ -189,17 +216,78 @@ function handleData([sheetsText, statesGeoJson]) {
     }
   });
 
+  countiesGeoJson = {
+    type: "FeatureCollection",
+    features: countiesCartoData.rows.map(
+      ({ the_geom, state, county }, index) => {
+        const properties = countiesData.filter(
```

Comment:
Follow up, I assume you did this because there may be more than one entry in the protections data for a given county. I've brought up data tidying with @erinmcel so hopefully we won't have to worry about duplicate entries and can avoid handling this client side, but it might be worth discussing further.